src/Utils.re

```diff
@@ -0,0 +1,37 @@
+module PxOrPrc = {
+  type t;
+  type arg =
+    | Px(int)
```

Comment:
I am a little bit afraid of this generic names. What will happen if there will be another package which defines same types?